os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -434,17 +435,27 @@ static void _up_assert(int errorcode)
 		}
 #endif
 	} else {
-		binmgr_mq = mq_open(BINMGR_REQUEST_MQ, O_WRONLY, 0666, 0);
-		DEBUGASSERT(binmgr_mq != (mqd_t)ERROR);
 
 		request_msg.cmd = BINMGR_FAULT;
 		request_msg.requester_pid = getpid();
+		if (current_regs) {
+			tcb = sched_self();
+			sched_removereadytorun(tcb);
+#if CONFIG_RR_INTERVAL > 0
+			tcb->timeslice = 0;
+#endif
+			tcb->sched_priority = SCHED_PRIORITY_MIN;
+			sched_addreadytorun(tcb);
+		}
 
 		/* Send a message to fault manager with pid of the faulty task */
-		ret = mq_send(binmgr_mq, (const char *)&request_msg, sizeof(binmgr_request_t), BINMGR_FAULT_PRIO);
+		ret = mq_send(gbinmgr_mq, (const char *)&request_msg, sizeof(binmgr_request_t), BINMGR_FAULT_PRIO);
```

Comment:
Please rename the variable g_binmgr_mq according to TizenRT convention.

Reply:
used common fd for both read and write

Reply:
changed to single fd for read and write